db/migrate/20170713183443_create_log_events.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLogEvents < ActiveRecord::Migration
+  def change
+    create_table :log_events do |t|
+      t.references :loggable, polymorphic: true
+      t.string :event_type
+      t.references :user, index: true, foreign_key: false
```

Comment:
Foreign key is false because none of the `created_by` or `updated_by` columns are foreign keys. Also, I want to minimize the chance this save causes an error